packages/website/docs/_samples/main/Table/Virtualizer/Virtualizer.md

```diff
@@ -0,0 +1,16 @@
+import html from '!!raw-loader!./sample.html';
+import js from '!!raw-loader!./main.js';
+
+Enhance your Table with virtualization capabilities by incorporating the **Virtualizer** feature.
+
+When dealing with virtualization in a table, the concept of `range-change` event with `first` and `last` parameters is crucial for understanding which rows are currently visible and need to be rendered. It is also essential to set specific attributes to ensure proper virtualization. These attributes are:
+
+1. `row-count` for the Table: This attribute specifies the total number of rows in the table. It helps the virtualizer understand how many rows it needs to manage.
```

Comment:
numbered list: I would tend  to use a bulleted list since this doesn't seem to be a procedure. Or is the order important?